controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -143,27 +143,59 @@ http {
         ''               close;
     }
 
+    {{ if (trustHTTPHeaders $cfg) }}
```

Comment:
For some reason, trustHTTPHeaders is always returning true for me. This block here is always being created regardless of whether I set real-client-from: "http-proxy" or real-client-from: "tcp-proxy".

I checked to make sure whether it's loading real-client-from properly by setting an invalid value, and it does seem to show up on the logs and sets to "auto" so I'm sure this value is being read up to trustHTTPHeaders.